README.md

```diff
@@ -105,16 +125,23 @@ fn main() {
 }
 ```
 
-Larger lifetimes can still shrink to unify with `'id`. What all this means is that you
-can't just trust any old `'id` lifetime floating around. It has to be carried in a
-trusted carrier; one that can't be created from an untrusted lifetime carrier.
+Larger variant lifetimes can still shrink to unify with `'id`. What all this
+means is that you can't just trust any old `'id` lifetime floating around. It
+has to be carried in a trusted carrier, and one that can't be created from an
+untrusted lifetime carrier.
```

Comment:
> and one that can't be created from an untrusted lifetime carrier.

I would just remove this extra condition because part of the contract of a "trusted carrier" is that it *was not* created from an untrusted lifetime, otherwise you have UB. When I see a trusted carrier it is safe to assume to was constructed properly.

 But if we want to keep this clause I would rephrase it as: 

> and one that wasn't created from an untrusted lifetime.

Because technically you *can* always create a carrier from an untrusted lifetime with unsafe code.